utils/firestore.js

```diff
@@ -453,6 +453,38 @@ const removeParticipantsDataDestruction = async () => {
     }
 }
 
+const removeUninvitedParticipants = async () => {
+    try {
+
+        const currSnapshot = await db
+            .collection('participants')
+            .where('288972510', '==', 353358909)
```

Comment:
I think we can replace the concept Ids for `288972510` and  `353358909` with a reference to [fieldToConceptIdMapping.js](https://github.com/episphere/connectFaas/blob/dev/utils/fieldToConceptIdMapping.js)

Reply:
Solved